src/main/java/creativei/manager/impl/InquiryManagerImpl.java

```diff
@@ -37,9 +37,15 @@ public ResponseObject getAll() {
 
     @Override
     public ResponseObject getByStatus(InquiryStatus status) {
-        List<Inquiry> inquiries = inquiryService.getByStatus(status);
-        List<InquiryVo> inquiryVos = ResponseHelper.getInquiryResponseByStatus(inquiries, new InquiryVo());
-        return ResponseObject.getResponse(inquiryVos);
+        List<Inquiry> inquiries = null;
+        try {
+            inquiries = inquiryService.getByStatus(status);
+            List<InquiryVo> inquiryVos = ResponseHelper.getInquiryResponseByStatus(inquiries, new InquiryVo());
```

Comment:
Why are we passing new InquiryVo object?

Reply:
the details we are sending to the response are of inquiryVo type so we need inquiryVo either we have to pass it in  the method or create within that method. This is the common approach we have done in all method where we pass inquiry and inquiryVo so i had pass the inquiryVo this wAY

Reply:
Helper is our class. It can have different method signatures based on our requirements. Methods should be optimized. Create vo inside the method and return

Reply:
okay sir

Reply:
Issue solved in commit 442cfec